rotkehlchen/exchanges/bittrex.py

```diff
@@ -274,6 +276,16 @@ def _single_api_query(
         except requests.exceptions.ConnectionError as e:
             raise RemoteError(f'Bittrex API request failed due to {str(e)}')
 
+        # Error due to unsync system clock (only for non-public endpoints)
+        if not public_endpoint and response.status_code == HTTPStatus.UNAUTHORIZED:
+            json_ret = rlk_jsonloads_dict(response.text)
+            if json_ret.get('code') == 'INVALID_TIMESTAMP':
+                current_time = datetime.fromtimestamp(int(api_timestamp) / 1000)
```

Comment:
I haven't replaced with 0 the milliseconds here as it may help letting us know in case we missed them and better logging info. But probably the request would fail because of bad data. I can remove them.

Reply:
I am not sure I understand what you are saying here. What request will fail because of bad data? Milliseconds are pushed into the datetime.

Reply:
My point was that adding the milliseconds in this exception won't ever help us in case we missed to convert the timestamp to milliseconds as the remote API will fail due to bad timestamp format. On the user side they are just noise in the exception message, and for us they doesn't add too much value either. This is why I'm wondering if just replace them with 0.

Reply:
Sure you can. No need to have them